artifactory/groups.go

```diff
@@ -54,11 +55,23 @@ func (s *GroupsService) GetAll() (*[]Group, *Response, error) {
 	return v, resp, err
 }
 
+// GetGroupRequest represents the possible inputs for a Get Group request.
+//
+// Docs: https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-GetGroupDetails
+type GetGroupRequest struct {
+	Name         *string
+	IncludeUsers *bool
+}
+
 // Get returns the provided group.
 //
 // Docs: https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-GetGroupDetails
-func (s *GroupsService) Get(group string) (*Group, *Response, error) {
-	u := fmt.Sprintf("/api/security/groups/%s", group)
+func (s *GroupsService) Get(groupRequest *GetGroupRequest) (*Group, *Response, error) {
+	u := fmt.Sprintf("/api/security/groups/%s", *groupRequest.Name)
+	if groupRequest.IncludeUsers != nil {
```

Comment:
I'm wondering if we should instead refactor this to only add `?includeUsers=<option>` if `groupRequest.IncludeUsers` == `true` to allow for compatibility older Artifactory servers. Thoughts?

Reply:
Your suggestion is probably the safest way to implement this, so I'd happily refactor.